Source/Plugins/NetworkLoggerPlugin.swift

```diff
@@ -25,14 +25,15 @@ public final class NetworkLoggerPlugin: PluginType {
 
     public func willSendRequest(_ request: RequestType, target: TargetType) {
         if let request = request as? CustomDebugStringConvertible , cURL {
+            output(request.debugDescription)
```

Comment:
@sunshinejr can you explain this line? It's causing an error for me. Is it a typo?